### DISCUSSION TITLE
Error while requesting lldp neighbors

### DISCUSSION THREAD

**Discussion by drziv** [netbox-community/netbox#10895] [Q&A] [answered]
Good afternoon. Please help me, what could be the problem when I request lldp networks through the browser, an error appears in the browser: No module named netmiko.ssh_exception.
What could I break?
napalm, netmiko installed. Everything worked before.
maybe this happened after renaming the "Device Types" for switches?

**Comment by candlerb** [comment, marked as answer]
I've seen this one before. If you [search in the forum](https://github.com/netbox-community/netbox/discussions?discussions_q=No+module+named+netmiko.ssh_exception) you'll find [the answer](https://github.com/netbox-community/netbox/discussions/9771).

**Comment by drziv** [comment]
Thanks a lot!!! I fixed everything/
I uninstalled napalm and netmiko
Installed napalm-ros. Netmiko was installed with him
it was like this:
(venv) root@NetBox:/opt/netbox# pip freeze | grep napalm
napalm==3.3.1
napalm-ros==1.2.2
(venv) root@NetBox:/opt/netbox# pip freeze | grep netmiko
netmiko==4.1.2

Now like this:
(venv) root@NetBox:/opt/netbox# pip freeze | grep napalm
napalm==4.0.0
napalm-ros==1.2.2
(venv) root@NetBox:/opt/netbox# pip freeze | grep netmiko
netmiko==4.1.2
